pkg/processor/runtime/python/py/_nuclio_wrapper.py

```diff
@@ -212,47 +223,30 @@ async def _initialize_context(self):
                 raise
 
     def _register_to_signal(self):
-        signal.signal(signal.SIGUSR1, self._on_termination_signal)
-        signal.signal(signal.SIGUSR2, self._on_drain_signal)
-
-    def _on_drain_signal(self, signal_number, frame):
-        self._logger.debug_with('Received signal, calling draining callback',
-                                signal=signal.Signals(signal_number).name)
+        on_termination_signal = functools.partial(self._on_termination_signal, "SIGUSR1")
+        on_drain_signal = functools.partial(self._on_drain_signal, "SIGUSR2")
 
-        if self._is_waiting_for_event:
-            self._logger.debug('Wrapper is waiting for an event, calling drain handler')
+        asyncio.get_running_loop().add_signal_handler(signal.SIGUSR1, on_termination_signal)
+        asyncio.get_running_loop().add_signal_handler(signal.SIGUSR2, on_drain_signal)
```

Comment:
For better maintenance - add the following the the `Constants` class:
```
termination_signal = signal.SIGUSR1
drain_signal = signal.SIGUSR2
```

Then use it here:
```python
        on_termination_signal = functools.partial(self._on_termination_signal, Constants.termination_signal.name)
        on_drain_signal = functools.partial(self._on_drain_signal, Constants.termination_signal.name)

        asyncio.get_running_loop().add_signal_handler(Constants.termination_signal, on_termination_signal)
        asyncio.get_running_loop().add_signal_handler(Constants.drain_signal.name, on_drain_signal)
```

Reply:
Sure, can do, but to be clear, direct use of `signal` module constants is not something that was introduced in this PR.

Reply:
Actually I'm not really sure I see the point in copying the constant from the signal package to `Constants`...

Reply:
My thought was that the signal itself will only be referenced in a one place.
Anyway, at least use `signal.SIGUSR1.name` instead of the plain string in the `partial` argument.

Reply:
Got it.